### ISSUE TITLE
Slow statistics in large data

### ISSUE THREAD

**Issue by hippke** [hippke/tls#66]
For short periods (many transits) and large data (Kepler K1), the statistics calculation takes a long time
Cause: pink_noise in def snr_stats
Solution: Disable for now?

**Comment by hippke** [comment]
solved by ``moving pinknoise = () `` in ``stats.py`` out of the main loop:
```
try:
    pinknoise = pink_noise(flux_ootr, int(numpy.mean(per_transit_count)))
except:
    pass

```

**Comment by hippke** [comment]
Fixed in 1.0.23
